pkg/setting/setting.go

```diff
@@ -1734,25 +1736,13 @@ func (cfg *Cfg) readRenderingSettings(iniFile *ini.File) error {
 }
 
 func (cfg *Cfg) readAlertingSettings(iniFile *ini.File) error {
+	// This check is kept to prevent users that upgrade to Grafana 11 with the legacy alerting enabled. This should prevent them from accidentally upgrading without migration to Unified Alerting.
 	alerting := iniFile.Section("alerting")
 	enabled, err := alerting.Key("enabled").Bool()
-	cfg.AlertingEnabled = nil
-	if err == nil {
-		cfg.AlertingEnabled = &enabled
+	if err == nil && enabled {
+		cfg.Logger.Error("Option [alerting].enabled cannot be true. Legacy alerting is no longer supported. Delete the option and use configuration '[unified_alerting].enabled' to enable Grafana Alerting and start from the scratch. Refer to documentation about migration to the new alerting (https://grafana.com/docs/grafana/latest/alerting/set-up/migrating-alerts)")
```

Comment:
@brendamuir can you please check if this message informative enough?

Reply:
cfg.Logger.Error("Option [alerting].enabled cannot be true. Legacy alerting is removed. It is no longer deployed, enhanced, or supported. Delete [alerting].enabled and use '[unified_alerting].enabled' to enable Grafana Alerting. For more information on migrating to Grafana Alerting, refer to the documentation. (https://grafana.com/docs/grafana/latest/alerting/set-up/migrating-alerts)")

Reply:
cfg.Logger.Error("Option [alerting].enabled cannot be true. Legacy Alerting is removed. It is no longer deployed, enhanced, or supported. Delete [alerting].enabled and use '[unified_alerting].enabled' to enable Grafana Alerting. For more information, refer to the documentation on upgrading to Grafana Alerting. (https://grafana.com/docs/grafana/latest/alerting/set-up/migrating-alerts)")

Reply:
- not sure about the first bit starting cfg.Logger - cannot be true. is that to appear in the message on the UI? do we need that bit?
- do we want to say delete xxx and use xxx to enable GA or would we rather say to upgrade to GA? not sure. 
- We use "upgrade" in the docs-so would prob stick to using that rather than migrating?

Reply:
This will appear in the Grafana logs when it starts up, it won't be shown in the UI.

Reply:
This will appear in the Grafana logs at start up, it won't be shown in the UI.

Reply:
Brenda , thanks for the recommendation! One thing I would like to change. I am not sure that the article will exist in G11 because migration from legacy to UA will not be supported. I will put link `https://grafana.com/docs/grafana/v10.4/alerting/set-up/migrating-alerts/` which does not exist yet but that will be the last verstion that offers migration.

Reply:
yes, ok. the plan is to still have a short one pager available just in case in the OSS docs - but add this one for now.